### ISSUE TITLE
Broken OpenSCAD conversion

### ISSUE THREAD

**Issue by Jip-Hop** [jscad/OpenJSCAD.org#263]
When converting a working OpenSCAD file like so: `openjscad working_openscad.scad -o broken_jscad.jscad` the output file won't work.

I get the following error: `ReferenceError: getParameterDefinitionsCLI is not defined`. Removing this block of code fixes this:
```
  var wrappedMain = main
  main = function(){
    var paramsDefinition = (typeof getParameterDefinitions !== 'undefined') ? getParameterDefinitions : undefined
    return wrappedMain(getParameterDefinitionsCLI(paramsDefinition, {}))
  }
```
OpenSCAD and OpenJSCAD files in zip, together with output STL files: [conversion_test.zip](https://github.com/jscad/OpenJSCAD.org/files/1011666/conversion_test.zip).

**Comment by kaosat-dev** [comment]
Hi again @Jip-Hop !
Thanks for the detailed feedback !
Would you mind splitting this up into two issues though ?
They are completely unrelated, and the I am almost done fixing the first one.
The second one on the other hand is way more complicated to trace :(

**Comment by kaosat-dev** [comment]
Thanks @Jip-Hop !

**Comment by kaosat-dev** [comment]
Fixed in #264 , now works correctly in CLI and web, good catch @Jip-Hop !
